src/Overview/Overview.js

```diff
@@ -20,59 +20,159 @@ const CardsWrapper = styled.div`
   justify-content: center;
 `;
 
-const FiltersContainer = styled.div`
+const ControlContainer = styled.div`
   display: grid;
-  grid-template-rows: auto;
-  grid-template-columns: auto 100px;
-  grid-gap: 1rem;
+  grid-template-rows: 1fr;
+  grid-template-columns: 1fr 1fr;
+  grid-template-areas: "itemone  itemtwo";
+  grid-gap: 30rem;
   justify-content: center;
+  margin-left: 20px;
+  margin-right: 20px;
 `;
 
 const Filter = styled.input`
   font-size: 23px;
 `;
 
+const SortDropDown = styled.ul`
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333;
+`;
+
+const StyledLi = styled.li`
+  float: left;
+`;
+
+const Dropbtn = styled.div`
+  display: inline-block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+`;
+
+const DropDownContent = styled.div`
+  display: none;
+  position: absolute;
+  background-color: #f9f9f9;
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+  z-index: 1;
+`;
+
+const DropDownLi = styled(StyledLi)`
+  display: inline-block;
+  &:hover {
+    background-color: red;
+  }
+  &:hover ${DropDownContent} {
+    display: block;
+  }
+`;
+
+const SubA = styled.a`
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+  text-align: left;
+  &:hover {
+    background-color: #f1f1f1;
+  }
+`;
+
 export function Overview({ data }) {
-  console.log('DATA in overview', data);
+  console.log("DATA in overview", data);
   const history = useHistory();
-  const [filter, setFilter] = useState('');
+  const [filter, setFilter] = useState("");
+  const [sortOrder, setSortOrder] = useState("None");
 
-  const filteredPokemons = useMemo(() => (
-    data.filter((pokemon) => pokemon.name.includes(filter.toLowerCase()))
-  ), [filter, data])
+  const filteredPokemons = useMemo(
+    () =>
+      data
+        .filter((pokemon) => pokemon.name.includes(filter.toLowerCase()))
+        .sort((a, b) => {
```

Comment:
Good solution!
You can read about build in function `localeCompare` which could be helpful in sorting strings.